core/cas-server-core-util-api/src/main/java/org/apereo/cas/util/HttpUtils.java

```diff
@@ -113,7 +114,13 @@ public static HttpResponse execute(final HttpExecutionRequest execution) {
                 builder.setProxy(proxy);
             }
             val client = builder.build();
+
             return client.execute(request);
+        } catch(final SSLHandshakeException e) {
+            val url = execution.getUrl();
+            val paramPos = url.indexOf("?");
+            val sanitizedUrl = paramPos > 0 ? url.substring(0, paramPos) : url;
+            LoggingUtils.error(LOGGER, "SSL error accessing: [" + sanitizedUrl + "]", e);
         } catch (final Exception e) {
```

Comment:
This is not really related to tests, but when I get a trust exception, I like to know what host it is hitting so I know what certificate is not trusted. One of the logout calls fails with a trust exception but the test still passes. In some of the CAS code that uses HttpUtils, there doesn't seem to be a way to be an easy way to turn off trust checking.

Reply:
You should write a test case to cover these lines as well.

Reply:
Added test